mirai-console/backend/mirai-console/src/plugin/description/PluginDescription.kt

```diff
@@ -108,6 +109,16 @@ public interface PluginDescription {
      */
     public val dependencies: Set<PluginDependency>
 
+    /**
+     * 此插件依赖的 Mirai Console 版本号. 为 null 时则为不限制版本. 通常建议至少限制使用同一个主版本号.
+     * 如开发时依赖该插件版本 2.15.0, 则将版本限制设置为 `[2.15.0, 2.16.0)`, 表示大于等于 `2.15.0`, 小于 `2.16.0`.
```

Comment:
```suggestion
     * 如开发时依赖该插件版本 2.15.0, 则将版本限制设置为 `[2.15.0, 3.0.0)`, 表示大于等于 `2.15.0`, 小于 `3.0.0`.
```

如果只写 `[2.15.0, 2.16.0)`, 那开发者每个版本都需要更新, 然而一般来说 console 不会有 breaking change 导致插件需要更新. 因此我觉得限制到 3.0 (潜在 breaking change) 以前就可以了, 这样开发者的维护付出会比较小.